staging/src/k8s.io/code-generator/cmd/validation-gen/validators/openapi.go

```diff
@@ -32,17 +32,22 @@ func InitOpenAPIDeclarativeValidator(_ *ValidatorConfig) DeclarativeValidator {
 type openAPIDeclarativeValidator struct{}
 
 const (
-	markerPrefix     = "k8s:validation:"
+	//FIXME: If we change this to "k8s:" then kube-openapi's parsing tries to
+	//       parse all tags, not just ones it knows. The previous tag is used
+	//       in some types.go fields, so just give it a bogus name for now.
+	markerPrefix     = "k8s:validation:new:"
```

Comment:
nitpit: need to update the comment below
```go
        var result Validations
	// Leverage the kube-openapi parser for 'k8s:validation:' validations.
	schema, err := generators.ParseCommentTags(t, comments, "+"+markerPrefix)
```

Reply:
Done in next push